linux_os/guide/system/apparmor/var_apparmor_mode.var

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+title: 'AppArmor profiles mode'
+
+description: |-
+    enforce - Set all AppArmor profiles to enforce mode
+    <br />complain - Set all AppArmor profiles to complain mode
```

Comment:
nitpick: I don't have strong opinion on that, but it seems more readable if the break line tag is at the end of the line instead of the beginning of the next line. In any case, I leave this on the authors preference.